cmd/run.go

```diff
@@ -319,8 +319,12 @@ func run(cmd *cobra.Command, args []string) {
 			for i, entry := range source.Entries {
 				html := formatHTMLEmail(&entry, config, tpl, destination)
 				var HTMLFile *os.File
-				if HTMLFile, err = os.Create(fmt.Sprintf("%s - %d.html", pipe, i)); err != nil {
-					fatalErr(fmt.Errorf("cannot open config.toml for writing"))
+				err := os.MkdirAll("./data", 0755)

```

Comment:
does this work on windows platforms ?

Reply:
Not 100% sure as I have not tested on windows but made a few changes that should make it more platform agnostic :)